pages/_section/photos/_gallery.vue

```diff
@@ -9,6 +9,9 @@
       v-if="page.slides.length > 0"
       class="l-wrap l-container l-container--xl"
     >
+      <h1 class="is-vishidden">
```

Comment:
what does this class do? I don't see it defined

Reply:
visibility hidden - good for SEO